tasks/libs/common/junit_upload_core.py

```diff
@@ -104,18 +105,22 @@ def get_flaky_from_test_output():
     Read the test output file generated by gotestsum which contains a list of json for each unit test.
     We catch all tests marked as flaky in source code: they contain a certain message in the output field.
     """
-    TEST_OUTPUT_FILES = ["test_output.json", "test_output_fast.json"]
+    TEST_OUTPUT_FILE = "module_test_output.json"
     FLAKE_MESSAGE = "flakytest: this is a known flaky test"
     test_output = []
-    for test_file in TEST_OUTPUT_FILES:
-        if os.path.isfile(test_file):
-            with open(test_file) as f:
+    flaky_tests = set()
+    for module in DEFAULT_MODULES:
+        if os.path.isfile(os.path.join(module, TEST_OUTPUT_FILE)):
+            with open(os.path.join(module, TEST_OUTPUT_FILE)) as f:
```

Comment:
```suggestion
    for module in DEFAULT_MODULES:
        test_file = Path(module, TEST_OUTPUT_FILE)
        if test_file.is_file():
            with test_file.open() as f:
```
In the last modifications I used more `pathlib` than `os.path`, you can maybe use it here, it's less verbose